contracts/IdentityFactory.sol

```diff
@@ -2,29 +2,14 @@ pragma solidity ^0.4.24;
 
 import "zos-lib/contracts/Initializable.sol";
 import "contracts/Identity.sol";
-import "contracts/IdentityRegistry.sol";
 
 
 contract IdentityFactory is Initializable {
-  event IdentityCreated(uint48 indexed centrifugeId, address identity);
-  address registry;
 
-  function initialize(address _registry) public initializer {
-    registry = _registry;
-  }
-
-  function createIdentity(uint48 _centrifugeId) public {
-    require(_centrifugeId != 0x0);
-    IdentityRegistry identityRegistry = IdentityRegistry(registry);
-    // Require that the centrifugeId is not already registered in the IdentityRegistry
-    require(identityRegistry.getIdentityByCentrifugeId(_centrifugeId) == 0x0);
-
-    Identity identity = new Identity(_centrifugeId);
+  event IdentityCreated(address indexed centrifugeId);
+  function createIdentity() public {
+    Identity identity = new Identity();
     identity.transferOwnership(msg.sender);
```

Comment:
If we call the factory with metaMask it should be fine. 

On the cent-node we would need an extra parameter for the transferOwnership. 
`msg.sender` would be the node address and not the address owned by the user.

Reply:
Do you agree? @rdinicut 
The alternative is two transaction:
1. The nodes create the identity.
2. The node transfers the ownership to an address of the user.

Reply:
for the MVP I think we can skip this and just set the owner to msg.sender even on the node. Let's keep it as simple as possible for now. Adding it later on will always be possible by adding another create method that allows you to pass in another argument.